arrow-optics/src/main/kotlin/arrow/optics/extensions/set/at/SetKAt.kt

```diff
@@ -0,0 +1,49 @@
+package arrow.optics.extensions.set.at
+
+import arrow.core.SetK
+import arrow.optics.PLens
+import arrow.optics.extensions.SetKAt
+import kotlin.Any
+import kotlin.Boolean
+import kotlin.Deprecated
+import kotlin.PublishedApi
+import kotlin.Suppress
+import kotlin.jvm.JvmName
+
+@JvmName("at")
+@Suppress(
+  "UNCHECKED_CAST",
+  "USELESS_CAST",
+  "EXTENSION_SHADOWED_BY_MEMBER",
+  "UNUSED_PARAMETER"
+)
+@Deprecated(
+  message = "arrow.optics.extensions package is being deprecated, and it will be removed in 0.13.",
+  level = DeprecationLevel.WARNING
+)
```

Comment:
Check out [this suggestion](https://github.com/arrow-kt/arrow-optics/pull/87#discussion_r557274345) made by @nomisRev to add a replacement here